db/dcrpg/pgblockchain.go

```diff
@@ -427,9 +427,10 @@ type DBInfo struct {
 // parameters. By default, duplicate row checks on insertion are enabled. See
 // NewChainDBWithCancel to enable context cancellation of running queries.
 func NewChainDB(dbi *DBInfo, params *chaincfg.Params,
-	stakeDB *stakedb.StakeDatabase, devPrefetch bool) (*ChainDB, error) {
+	stakeDB *stakedb.StakeDatabase, devPrefetch, isHidePGConfig bool) (*ChainDB, error) {
```

Comment:
Just `hidePGConfig` is fine since `hide` is a verb indicating the action/state when it is true.